src/Modules/Admin/Commands.Admin/help/Get-PowerBIActivityEvent.md

```diff
@@ -73,10 +77,11 @@ Specifies the type of result that is returned by the cmdlet.
 Type: OutputType
 Parameter Sets: (All)
 Aliases:
+Accepted values: JsonString, JsonObject
 
 Required: False
 Position: Named
-Default value: JsonString
+Default value: None
```

Comment:
is this ok? Since the default is actually JsonString if the user doesnt specify the result type explicitly

Reply:
I fixed